pkg/notifiers/slack/handle_request.go

```diff
@@ -149,18 +149,21 @@ func (n *SlackNotifier) HandleRequestEvent(ctx context.Context, log *zap.Sugared
 				wg.Wait()
 			}
 		} else {
+
 			//Review not required
-			msg := fmt.Sprintf(":white_check_mark: Your request to access *%s* has been automatically approved.", requestedRule.Name)
+			msg := fmt.Sprintf(":white_check_mark: Your request to access *%s* in account has been automatically approved.\n", requestedRule.Name)
```

Comment:
I think we can omit the name of the rule here and instead add it to the table?
What do you think, given that the arguments are the most important information, saying that the request to <rule name> doesn't tell the user much

```suggestion
			msg := ":white_check_mark: Your request has been automatically approved.\n"
```

Reply:
I don't see any issue with having it in the default text, we could additionally add it to the table?